wrappers/wrapper.rb

```diff
@@ -572,6 +573,88 @@ def detect_unfeasible_services(vrp)
       unfeasible
     end
 
+    def sequence_relation_with_no_reachable_vehicle?(unfeasible, vrp, relation)
+      return false unless Models::Relation::POSITION_TYPES.include?(relation.type)
+
+      vrp.vehicles.each{ |v|
+        unreachable_service_id = last_service_reachable_sequence(vrp, v, relation.linked_services)
+        next unless unreachable_service_id
+
+        service_met = false
+        relation.linked_services.each{ |service|
+          service_met = true if service.id == unreachable_service_id
```

Comment:
This condition should be as follows no? Otherwise, we mark only the unreachable_service but not the services "following" it. 
```suggestion
          service_met ||= true if service.id == unreachable_service_id
```

Reply:
service_met stays true from the service with the unreachable_service_id on a given vehicle
It is reset to false only if we change the vehicle. `||=` may only avoid a re-affectation. It won't change the value.